### DISCUSSION TITLE
Wazuh Agent Configuration

### DISCUSSION THREAD

**Discussion by MHamza-07** [wazuh/wazuh#13866] [Q&A] [answered]
Hi Team !!

I am trying to configure Wazuh Agent on Solaris 11.3 by following this link:
https://documentation.wazuh.com/4.2/user-manual/reference/ossec-conf/client.html#reference-ossec-client-config-profile
I did the server configuration in client tag of file ossec.conf. However, I am unable to Configure the connection parameters for the agent enrollment since, i cannot find the client tag for enrollment.

![image](https://user-images.githubusercontent.com/85833913/173524949-d4f45438-42f2-430e-af0d-06bb91deba3d.png)

This is suppose to be done in file ossec.conf

Your kind response is awaited.
Thank You

**Comment by mdiego92** [comment, marked as answer]
Hello MHamza-07,

First of all, we would like to confirm if you have already downloaded and installed Wazuh Agent for Solaris 11.
[https://documentation.wazuh.com/4.2/installation-guide/wazuh-agent/wazuh-agent-package-solaris.html](https://documentation.wazuh.com/4.2/installation-guide/wazuh-agent/wazuh-agent-package-solaris.html
)

Regarding the <client> tag, you could add this option in the configuration file (/var/ossec/etc/ossec.conf) between the <ossec_config> tags:
```
<client>
  <enrollment>
    <enabled>yes</enabled>
    .
    .
    .
    <agent_name>AGENT NAME HERE</agent_name>
  </enrollment>
</client>
```

Finally, please restart the agent to apply the changes.

For further information about the agent registration you could check the following sections of our documentation:
[agent-enrollment](https://documentation.wazuh.com/current/user-manual/agent-enrollment/index.html)
[ossec-conf: client](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/client.html#agent-name)

Please let us know if you have any remaining questions or doubts.

Thank you very much!
